src/main/scala/strawman/collection/StringOps.scala

```diff
@@ -10,11 +10,14 @@ import scala.reflect.ClassTag
 
 class StringOps(val s: String)
   extends AnyVal
-     with SeqOps[Char, Seq, String]
-     with Buildable[Char, String]
-     with ArrayLike[Char] {
+    with IterableOnce[Char]
```

Comment:
I don't think `StringOps` should be a "proper" collection type (i.e. one that implements a trait in the basic IterableOnce -> Iterable -> ... hierarchy). Every method you add here is available on all Strings via an implicit conversion in Predef. We already have a second one for WrappedString (2.12) or StringView (strawman) which is an Iterable. By having only one such conversion in Predef you can easily opt out of the implicit conversion from String to a collection but still keep the StringOps methods.